tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py

```diff
@@ -127,7 +127,7 @@ def extra_configuration_after_yield():
 
 
 # Tests
-
+@pytest.mark.skip('Test skipped for flaky behavior, after it is fixed, it will be unblocked')
```

Comment:
Add the related issue URL here

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3784/commits/c13d163a8a5a4afd2688b3c923c69829fff7ce5f